Pod/Classes/HairlineView/README.md

```diff
@@ -0,0 +1,48 @@
+# HairlineView
+
+A UIView that can be defined by axis, thickness and color
+
+## Default Appearance
+
+<details>
+<summary>Screenshots</summary>
+
+<p>
```

Comment:
#? why the paragraph tags? Doesn't GitHub Markdown automatically add these when you do `\n\n`?

Reply:
I think it had something to do with keeping multiple screenshots side-by-side in the Acknowledgements readme. Left here as a convenience in case we added more screenshots, but I'm cool to remove.